README.md

```diff
@@ -31,13 +31,12 @@
 
 # 📖 [Moblie Portfolio] <a name="Moblie Portfolio"></a>
 
->The first project of Micriverse program .
+
 
 **[your_project__name]** is a Moblie Portfolio.
 
```

Comment:
- [ ] Kindly consider your readme to be more descriptive ,a descriptive readme says much about the project even without asking about the project like brief description of the project.

Reply:
> - [ ] As requested by the previous code reviewer, Please kindly a brief description of the project. This will help others who want to use your website understand its purpose.👍